bin/tx-prover/README.md

```diff
@@ -110,9 +110,39 @@ The worker service implements the [gRPC Health Check](https://grpc.io/docs/guide
 
 The proxy service uses this health check to determine if a worker is available to receive requests. If a worker is not available, it will be removed from the set of workers that the proxy can use to send requests, and will persist this change in the configuration file.
 
-## Logging
+## Logging and Tracing
 
-Both the worker and the proxy will use the `info` log level by default, but it can be changed by setting the `RUST_LOG` environment variable.
+The service uses the [`tracing`](https://docs.rs/tracing/latest/tracing/) crate for both logging and distributed tracing, providing structured, high-performance logs and trace data. By default, logs are written to `stdout`, and traces are exported to a Jaeger instance via OpenTelemetry.
+
+### Configuring Logging
+
+- **Default Log Level**: Both the worker and proxy use the `info` log level by default.
+- **Custom Log Levels**: You can configure log levels by setting the `RUST_LOG` environment variable. For example:
+    ```
+    export RUST_LOG=debug
+    ```
+- **Log Destination**: Logs are output to `stdout` by default.
+
+### Configuring Tracing
+
+- **Trace Exporter**: Traces are exported to a Jaeger instance using the OpenTelemetry protocol. The Jaeger instance can be run locally using Docker (see below).
```

Comment:
I'm still not quite clear on how this works. I'm assuming the service sends traces some local port - right? Does it have to be a Jaeger instance (i.e., there is no mentioning of Jaeger when we configure OpenTelemetry tracing)? How does it know which instance to use? If we have 2 Jaeger instances, can we specify somehow which one to use? Does it always have to be Jaeger or can we use some other OpenTelemetry subscriber?

And what if the instance is not there? Would the service somehow detect this and discard all the traces?